Java/PoolAndResourceFile/CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+# Contributing to Azure samples
```

Comment:
Shouldn't this also be up in at least the Java directory, if not the root directory?